docs/installation_and_administration_manual/install.md

```diff
@@ -1,5 +1,13 @@
 # Install Meteoroid
 
+## Install enviroment
```

Comment:
I think [Requirements] is a good title. 
And could you add Software Requirements? (e.g. Docker Version, Docker Compose Version)